src/data/real/ennreal.lean

```diff
@@ -111,7 +111,7 @@ protected def to_nnreal : ℝ≥0∞ → ℝ≥0
 protected def to_real (a : ℝ≥0∞) : real := coe (a.to_nnreal)
 
 /-- `of_real x` returns `x` if it is nonnegative, `0` otherwise. -/
-protected def of_real (r : real) : ℝ≥0∞ := coe (nnreal.of_real r)
+protected def of_real (r : real) : ℝ≥0∞ := coe (real.to_nnreal r)
```

Comment:
Should this be `real.to_ennreal`?

Reply:
That's the plan, but I only did the `nnreal` one in this PR to keep it at a reasonable size. I'll do the `ennreal` version once #7766 is merged, as I don't want to have too many merge conflicts.